app.js

```diff
@@ -0,0 +1,92 @@
+import { productsArray } from './productsArray.js';
+import { incrementTimesSeen } from './utilities.js';
```

Comment:
You can combine multiple named imports from the same file like this:
import { function1, function2, function3 } from './common-file.js'